CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file
 
+## [1.1.0] - 2020-04-16
```

Comment:
Same comment applies here about this being a major version bump since the functionality is changing but the interface is not: (https://github.com/Invoca/aggregate/pull/49#discussion_r413181152)

Thoughts?

Reply:
I actually kind of disagree here. Isn't the idea of this change to support backwards compatible filtering with and without millisecond timestamps?